lib/pbench/cli/agent/commands/generate_token.py

```diff
@@ -27,9 +27,7 @@ def execute(self):
         try:
             response = requests.post(f"{server}/login", headers=headers, json=payload)
         except requests.exceptions.ConnectionError as exc:
-            if hasattr(exc.args[0], "reason"):
-                raise exc.args[0].reason
-            raise
+            raise RuntimeError(f"Cannot connect to {server}") from exc
```

Comment:
Definitely an improvement, although I'd prefer to either show the full URI (including the `/login`) or else drop the path entirely and show just the host part (`http://pbench.example.com`). Nobody _actually_ connects to `http://pbench.example.com/api/v1` and you won't get any useful result if you try.

Reply:
But I should be able to test by using the full URI (with the `/login` part), right? If so, I think the complete URI should be reported. Or if that's not the case,  the "thing" that I could cut and paste into a browser to check the outcome.

Reply:
It now prints the full URI in the error message (see the PR Description).